src/compwa_policy/check_dev_files/ruff.py

```diff
@@ -574,6 +582,44 @@ def _update_precommit_hook(precommit: ModifiablePrecommit, has_notebooks: bool)
     precommit.update_single_hook_repo(expected_repo)
 
 
+def _sort_imports_on_top(
+    precommit: ModifiablePrecommit, pyproject: ModifiablePyproject
+) -> None:
+    __add_isort_configuration(pyproject)
+    __add_nbqa_isort_pre_commit(precommit)
+
+
+def __add_isort_configuration(pyproject: ModifiablePyproject) -> None:
+    isort_settings = pyproject.get_table("tool.isort", create=True)
+    minimal_settings = dict(
+        profile="black",
+    )
+    if not complies_with_subset(isort_settings, minimal_settings):
+        isort_settings.update(minimal_settings)
+        msg = "Made isort configuration compatible with Ruff"
+        pyproject.changelog.append(msg)
+
+
+def __add_nbqa_isort_pre_commit(precommit: ModifiablePrecommit) -> None:
+    existing_repo = precommit.find_repo("https://github.com/nbQA-dev/nbQA")
+    excludes = None
+    if existing_repo is not None and existing_repo.get("hooks"):
+        nbqa_hook_candidates = [
+            h for h in existing_repo["hooks"] if h["id"] == "nbqa-isort"
+        ]
+        if nbqa_hook_candidates:
+            nbqa_hook = nbqa_hook_candidates[0]
+            excludes = nbqa_hook.get("exclude")
+    expected_repo = Repo(
+        repo="https://github.com/nbQA-dev/nbQA",
+        rev="1.9.0",
+        hooks=[Hook(id="nbqa-isort", args=YAML(typ="rt").load("[--float-to-top]"))],
```

Comment:
rt?

Reply:
Stands for "round trip"
https://yaml.readthedocs.io/en/latest/detail/#round-trip-including-comments

But you have a point, code could be extracted to some function that has an explanatory name.